plugins/ground_plane_editor.js

```diff
@@ -1,222 +1,251 @@
+        icon,
+        author,
+        description: "Edits the opacity and color of the ground plane feature in Blockbench.",
+        about: "This simple plugin allows you to customize the ground plane feature in Blockbench; more specifically, the opacity and color.\n## How to use\nTo use this plugin, simply go to `Tools > Ground Plane Editor`, fill out the appropriate categories, and hit `Done`. You can choose to edit either the color, the opacity, or both!\n\nPlease report any bugs or suggestions you may have.",
+        tags: ["Ground Plane", "Animation", "Customization"],
+        version: "1.1.0",
+        min_version: "4.2.0",
+        variant: "both",
+
+        oninstall() {
+            showAbout(true)
+            Blockbench.showQuickMessage("Successfully Installed Ground Plane Editor!");
+        },
+        onuninstall() {
+            Blockbench.showQuickMessage("Uninstalled Ground Plane Editor");
+        },
+        onload() {
+            addAbout()
+            action = new Action({
+                id,
+                name: "Edit Ground Plane",
+                icon,
+                condition: () => Format?.id !== "image",
+                click() {
+                    const timeout = {
+                        example: null
+                    }
+
+                    // Dialog to edit ground plane
+                    const dialog = new Dialog({
+                        title: "Edit Ground Plane",
+                        id: "edit_ground_plane_dialog",
+                        lines: [`
+                            <li></li>
+                            <style>
+                                dialog#edit_ground_plane_dialog .bar {
+                                    display: flex;
+                                    align-items: center;
+                                    margin: 0!important;
+                                    height: 30px;
+                                    box-sizing: content-box;
+                                    overflow: hidden;
+                                }
+
+                                dialog#edit_ground_plane_dialog input[type=range] {
+                                    flex-grow: 1;
+                                    margin-left: 50px;
+                                }
+
+                                dialog#edit_ground_plane_dialog input[type=number] {
+                                    margin: 0 8px 0 2px;
+                                }
+                            </style>
+                        `],
+                        component: {
+
+                            // Method runs when the dialog is opened
+                            mounted() {
+                                this.$nextTick().then(() => {
+                                    let opacityScaled = Math.floor(Canvas.ground_plane.material.opacity * 255);
+                                    $("dialog#edit_ground_plane_dialog .slider_input_combo #opacity_number").val(opacityScaled);
```

Comment:
I'm not 100% sure on what this implements that I don't currently have in my plugin. Could you elaborate on that? (I probably missed something so my apologies)

Reply:
You are currently using jQuery to set and read the values from the input. It would be much easier to use v-model.

You could also use a form by the way.

Reply:
My current system does work fine. Unless you really feel like I should change, I don't think it's worth it for now.